Denik CZ.js

```diff
@@ -0,0 +1,131 @@
+{
+	"translatorID": "c5b2b25d-5a0f-4e90-b8e7-4be4931a64f5",
+	"label": "Denik",
+	"creator": "Jiří Sedláček - Frettie",
+	"target": "",
```

Comment:
Target has to be a regexp that matches URL of the target pages. It should be as restrictive as possible, in the form `"^https?://<something that matches the domain here>/<possibly more stuff here>"`